qldpc/abstract.py

```diff
@@ -14,43 +14,43 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-All groups in this module are finite, and represented under the hood as a sympy PermutationGroup, or
-a subgroup of the symmetric group.  Group members are essentially represented by sympy Permutation
+All groups in this module are finite, and represented under the hood as a Sympy PermutationGroup, or
+a subgroup of the symmetric group.  Group members are essentially represented by Sympy Permutation
 objects.  Groups additionally come equipped with a representation, or "lift", that maps group
 elements to square matrices, such that the group action gets lifted to matrix multiplication.
 
 !!! WARNINGS !!!
 
 Whereas matrices are "left-acting" (that is, act on objects from the left) by standard convention,
-sympy permutations are "right-acting", which is to say that the action of two permutations p and q
+Sympy permutations are "right-acting", which is to say that the action of two permutations p and q
 on an integer i compose as (p*q)(i) = q(p(i)) = i^p^q.  To preserve the order of products before and
 after lifting permutations to matrices, which ensures that the lift L(p*q) = L(p) @ L(q), we
 therefore make representations likewise right-acting, which is to say that a permutation matrix M
 transposes a vector v as v --> v @ M.  In practice, this simply means that matrices are the
 transpose of what one might expect.
 
-This module only supports unitary representations of group members by integer-valued square
-matrices.  The restriction to unitary representations is not fundamental, but it is very convenient
-for identifying the "transpose" a group member p with respect to a representation (lift) L.  This
-transpose is defined as the group member p.T for which L(p.T) = L(p).T: if the representation is
-unitary, then p.T is equal to the inverse ~p = p**-1.
+This module only supports representations of group members by orthogonal matrices over finite
```

Comment:
"orthogonal matrices over finite fields" in abstract.py -- Aren't the lifts always integer-valued permutation matrices?

Reply:
Only by default.  Users are still free to provide other lifts, and I can imagine adding groups with other lifts in the future.  The `QuaternionGroup` currently in `abstract.py` is one (perhaps not-too-interesting) example.